### PR TITLE
Test htmltest IgnoreURL fix

### PR DESCRIPTION
**Pull request by yzdocs**

### CODE REVIEW

File: .htmltest_nightly.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ IgnoreDirs:
   - ^docs/v1.2
 IgnoreURLs:
   - http://localhost:4000
-  - https://github.com
+  - "https://github.com"
```
Comment (jseldess) [inline .htmltest_nightly.yml:9]: Based on the example in the readme, this looks like the right format now: https://github.com/wjdp/htmltest#example

Comment (yzdocs) [reply .htmltest_nightly.yml:9]: I also had to pull in a fix to our fork of `htmltest`: https://github.com/cockroachdb/htmltest/commit/e8e1406f1f25ac02c372da23f0cf50a09d77abfc

Testing now to see if that worked.

Comment (yzdocs) [reply .htmltest_nightly.yml:9]: I think the Netlify build cache may be preventing the `htmltest` changes I mentioned above from being applied.

Comment (yzdocs) [reply .htmltest_nightly.yml]: I've ruled out the build cache as a factor. It seems that the underlying code in `htmltest` isn't working as expected, even with the fix I mentioned trying to pull in above.

Comment (yzdocs) [reply .htmltest_nightly.yml]: Nevermind, I blundered! Will update soon.

**Pull request merged**